BookingsAPI/Bookings.UnitTests/Controllers/HearingsControllerTest.cs

```diff
@@ -82,10 +84,67 @@ public async Task should_return_next_and_previous_page_urls()
             var result = await _controller.GetHearingsByTypes(caseTypes, "0", 2);
 
             result.Should().NotBeNull();
-            var response = (BookingsResponse)((ObjectResult) result.Result).Value;
+            var response = (BookingsResponse)((ObjectResult)result.Result).Value;
             response.PrevPageUrl.Should().Be("hearings/types?types=1&cursor=0&limit=2");
             response.NextPageUrl.Should().Be("hearings/types?types=1&cursor=next-cursor&limit=2");
+        }
+
+        [Test]
+        public async Task should_change_hearing_status_to_created_and_send_event_notification()
+        {
+            var request = new UpdateBookingStatusRequest
+            {
+                UpdatedBy = "email@toupdate.com",
+                Status = Api.Contract.Requests.Enums.UpdateBookingStatus.Created
+            };
+            var hearingId = Guid.NewGuid();
+            var hearing = GetHearing();
+
+            _queryHandlerMock
+             .Setup(x => x.Handle<GetHearingByIdQuery, VideoHearing>(It.IsAny<GetHearingByIdQuery>()))
+             .ReturnsAsync(hearing);
+
+            var result = await _controller.UpdateBookingStatus(hearingId, request);
+
+            result.Should().NotBeNull();
+            var objectResult = (NoContentResult)result;
+            objectResult.StatusCode.Should().Be((int)HttpStatusCode.NoContent);
+        }
+
+        [Test]
+        public async Task should_change_hearing_status_to_cancelled()
+        {
+            var request = new UpdateBookingStatusRequest
+            {
+                UpdatedBy = "email@toupdate.com",
+                Status = Api.Contract.Requests.Enums.UpdateBookingStatus.Cancelled
+            };
+            var hearingId = Guid.NewGuid();
+            var hearing = GetHearing();
+
+            _queryHandlerMock
+             .Setup(x => x.Handle<GetHearingByIdQuery, VideoHearing>(It.IsAny<GetHearingByIdQuery>()))
+             .ReturnsAsync(hearing);
+
+
+            var result = await _controller.UpdateBookingStatus(hearingId, request);
+
+            result.Should().NotBeNull();
+            var objectResult = (NoContentResult)result;
+            objectResult.StatusCode.Should().Be((int)HttpStatusCode.NoContent);
```

Comment:
you haven't verified the cancelled event was actually sent, read from the service bus queue client